src/CgStr.hpp

```diff
@@ -6,13 +6,14 @@
 #include <cassert>
 #include <ostream>
 #include <string>
+#include <utility>
 
 namespace synth {
 
 class CgStr {
 public:
     CgStr(CXString&& s)
-        : m_data(s)
+        : m_data(std::move(s))
```

Comment:
Since CXString comes from a C library, it is a POD. But nonetheless moving is OK as documentation. :+1: